src/Components/Clock/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './Clock.scss';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  date: string,
+};
+
+class Clock extends React.Component<Props, State> {
+  state = {
+    date: '',
+  };
+
+  timerId?: NodeJS.Timer;
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      this.setState({
+        date: new Date().toLocaleTimeString(),
+      });
+      // eslint-disable-next-line
+      console.log(this.state.date);
+    }, 1000);
+  }
+
+  componentDidUpdate({ name }: Readonly<Props>) {
+    if (this.props.name !== name) {
+      // eslint-disable-next-line
+    console.log(`The Clock was renamed from ${name} to ${this.props.name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.timerId) {
+      clearInterval(this.timerId);
+    }
+  }
+
+  render() {
+    const { date } = this.state;
+
+    return (
+      <div className="clock">
+        <p className="clock-info">Current time:</p>
```

Comment:
If we have scss file, then I think you should rename class with nesting: for example clock__info. And use nesting in scss file.